protocols/everclear.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "Everclear",
+    "socials": [
+        {
+            "label": "App",
+            "link": "https://bridge.connext.network/"
+        },
+        {
+            "label": "Website",
+            "link": "https://everclear.org/"
+        },
+        {
+            "label": "Docs",
+            "link": "https://docs.everclear.org/"
+        },
+        {
+            "label": "Twitter",
+            "link": "https://twitter.com/EverclearOrg/"
+        },
+        {
+            "label": "Discord",
+            "link": "https://discord.gg/everclear"
+        },
+        {
+            "label": "Explorer",
+            "link": "https://connextscan.io/"
+        },
+        {
+            "label": "GitHub",
+            "link": "https://github.com/connext"
+        }
+    ],
+    "logo_dark": "/protocols/everclear-dark.png",
+    "logo_light": "/protocols/everclear-light.png",
+    "icon_url": "/protocols/everclear-dark.png",
+    "icon_dark": "/protocols/everclear-dark.png",
+    "icon_light": "/protocols/everclear-light.png",
+    "description": "Everclear is a novel public good mechanism for settling intents across chains.",
+    "mainnet": "yes",
+    "type": "bridge",
+    "chains_supported": "Enables permissionless expansion to new chains and rollups, making it possible for any application on any chain to onboard users from any other chain.",
+    "xchain_sync": "Canonical brdiges for message passing",
```

Comment:
spelling

Reply:
Also is this true?

Reply:
> "xchain_sync": "Canonical brdiges for message passing", 
oh, yes, I forgot to ask what is this field anyways? I didn't find anything useful by googling. I just copied that one from the other protocols. I also didn't find it used anywhere on the intent.markets, do we even need this field?

Reply:
typo fixed, if it's true or not I don't know yet